modules/sdk/gradle-plugins-workspace/src/gradleTest/verifyCustomBundleChecksumMD5Skip/gradle.properties

```diff
@@ -0,0 +1 @@
+liferay.workspace.product = portal-7.3-ga3
```

Comment:
you should use a customized bundle url(like https://releases.liferay.com/portal/7.3.3-ga4/liferay-ce-portal-tomcat-7.3.3-ga4-20200701015330959.tar.gz) to verify this test

Reply:
If the gradle workspace plugin found the bundle url is different from default bundle defined in product info, then verifyTask will be skipped. Please follow this logic.

Reply:
Due to the liferay.workspace.bundle.url was defined in build.gradle and added command line argument (--project-prop) to gradleTest task, it perhaps lead to the properties defined in gradle.properties may be ignored.So I was wondering whether to delete gradleArguments in gradleTest block which was defined in build.gradle.